### ISSUE TITLE
EE: Create commands do not work

### ISSUE THREAD

**Issue by Zignature** [mindpixel-labs/vsc-expressionengine#16]
### Prerequisites

* Have you tried to reproduce the problem in with the extension disabled?
     * This is not possible/applicable

* Have you checked that your issue isn't already filed?
     * It's not filed yet

### Description

`EE: Create` commands do not work.

### Steps to Reproduce

1. Open command palette with [CTRL] + [SHIFT] + p
2. Type `ee create` in the command palette
3. Select `EE: Create Add-on Boilerplate`
3. Select any option (Extension, Module or Plugin)
3. Enter my vendor name
3. Enter my add-on name

**Expected behavior:**

Expected folder(s) and files to be created and the boilerplate to open in the editor.

**Actual behavior:**

In the case of creating an add-on nothing, really nothing, happens. No error message and no folder(s) and files are created. Subsequently no boilerplate opens in the editor.

In the case of the other commands nothing happens other than an error message appearing.

**Reproduces how often:**

100%. All the time.

### Versions

* vsc-expressionengine: v2.1.3
* OS: Windows 10 Pro
* OS version: 21H1 build 19043.1348

### Additional Information

I edited all the necessary settings as per the README file.

When trying to create a template an error message appeared and it pointed out the issue.
My workspace is located at `D:\webdev\EE-Dev\sites\ee6u\` and the extension prefixes the workspace path with `C:\`.

The error message:

```
The template could not be created: Error: EINVAL: invalid argument, mkdir 'C:\d:\webdev\EE-Dev\sites\ee6u\system\user\templates\default_site\blog.group'
```

### Solution (temporary)
Use a hardcoded path for the `Ee: User Path` setting (e.g.  `"ee.userPath" : "D:/webdev/EE-Dev/sites/ee6u/system/user"` ). 
This works fine for the time being.

**Comment by mindpixel-labs** [comment]
Thanks for reporting this @Zignature I will review!

**Comment by mindpixel-labs** [comment]
@Zignature It appears to be a bug on Windows 10 where NodeJS is including a duplicate `C:\` in the path upon using any of the create commands. You can for now continue to user the `ee.userPath` override variable for now but I will patch this soon. Thanks again for reporting this issue.

**Comment by Zignature** [comment]
@mindpixel-labs you're welcome :)

**Comment by mindpixel-labs** [comment]
@Zignature Hope you are doing well. Sorry for the long delay and no update on this. I am finally preparing a new release soon that will squash this bug on Windows and some other improvements. Should be dropping in the coming weeks.

Thanks!

**Comment by mindpixel-labs** [comment]
Marking as closed, this is now fixed in V3.0.0. Thanks!
